### ISSUE TITLE
Upgrade F1-micro to E2-Micro by Aug. 31, 2021

### ISSUE THREAD

**Issue by nickFalcone** [rajannpatel/Pi-Hole-on-Google-Compute-Engine-Free-Tier-with-Full-Tunnel-and-Split-Tunnel-Wireguard-VPN-Configs#45]
I received an email from GCP instructing me to change my F1-micro instance to E2-micro by August 31, 2021 to continue using the free tier. The E2-micro free tier becomes available on August 1, 2021. 

I will attempt to change over on August 1st. I opened this issue to track documentation updates in [GOOGLE-CLOUD.md](https://github.com/rajannpatel/Pi-Hole-on-Google-Compute-Engine-Free-Tier-with-Full-Tunnel-and-Split-Tunnel-Wireguard-VPN-Configs/blob/master/GOOGLE-CLOUD.md) where there are references to the F1-micro VM.

Email excerpt below:

> **What do I need to know?**
> The Free Tier F1-micro VM is changing to the E2-micro VM as the VM to use for free. On August 1, 2021, E2-micro Free Tier will be introduced. Follow [these steps](https://notifications.google.com/g/p/AD-FnEwRLJ_ZOXuTfpl0O92oNGHNwnTbAV3vpugrDPMiIhy8jCnpsvXaLd9lflZqeYBpgkoSAaUPESlXsGeL62CVkczhAjTUzJWCNbRG7vlIL6Xc427jWa-fZZTI1k9-enNfUA7zgGHo8BYjarXHuUtZYK6BFBJS6Ht9lDOMpt0IA71NIjARHGn1Zy1duzDJLOM) to change your machine type to E2-micro to avoid incurring charges for continuing to use F1-micro after August 31, 2021.
> 
> **What do I need to do?**
> Starting August 1, 2021, [change the machine type](https://notifications.google.com/g/p/AD-FnEwRLJ_ZOXuTfpl0O92oNGHNwnTbAV3vpugrDPMiIhy8jCnpsvXaLd9lflZqeYBpgkoSAaUPESlXsGeL62CVkczhAjTUzJWCNbRG7vlIL6Xc427jWa-fZZTI1k9-enNfUA7zgGHo8BYjarXHuUtZYK6BFBJS6Ht9lDOMpt0IA71NIjARHGn1Zy1duzDJLOM) from F1-micro, or stop your existing F1-micro instance and begin using a E2-micro instance. VMs created with either method will automatically have the Free Tier discount applied to them. The supported regions will remain the same.

**Comment by Kwbmm** [comment]
+1 Received same mail. Should be specified in the documentation

**Comment by nickFalcone** [comment]
I updated my instance today from F1-micro to E2-micro with no issues by following [change machine type GCP docs](https://cloud.google.com/compute/docs/instances/changing-machine-type-of-stopped-instance#changing_a_machine_type).

![e2Micro](https://user-images.githubusercontent.com/6636483/128606029-d4888645-9b8f-43f4-8988-0454628cd370.JPG)

Pi-Hole admin dashboard and output from these two test sites look good after reactivating WireGuard:

- https://www.dnsleaktest.com/
- https://blockads.fivefilters.org/

**Comment by jcat59** [comment]
FYI - I followed the guide and my only change was to use the E2 instance based on this comment. Everything worked great.

As a side note, I accidentally chose the 'balanced persistent disk' rather than the 'standard persistent disk', and realized after I started getting charges. Had to figure out how to stop the VM, make a snapshot of the disk, move the VM to the new disk, then delete the old disk. Everything back up and running now.
